run/GCHP/HISTORY.rc.templates/HISTORY.rc.fullchem

```diff
@@ -5011,7 +5011,142 @@ COLLECTIONS: @#'DefaultCollection',
+                            'SpeciesConcVV_OCIL07          ', 'GCHPchem',
+                            'SpeciesConcVV_OCIL08          ', 'GCHPchem',
+                            'SpeciesConcVV_OCIL09          ', 'GCHPchem',
+                            'SpeciesConcVV_OCIL10          ', 'GCHPchem',
+                            'SpeciesConcVV_OCIL11          ', 'GCHPchem',
+                            'SpeciesConcVV_OCIL12          ', 'GCHPchem',
+                            'SpeciesConcVV_OCIL13          ', 'GCHPchem',
+                            'SpeciesConcVV_OCIL14          ', 'GCHPchem',
+                            'SpeciesConcVV_OCIL15          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST01          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST02          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST03          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST04          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST05          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST06          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST07          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST08          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST09          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST10          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST11          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST12          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST13          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST14          ', 'GCHPchem',
+                            'SpeciesConcVV_DUST15          ', 'GCHPchem',
+                            'SpeciesConcVV_AW01          ', 'GCHPchem',
+                            'SpeciesConcVV_AW02          ', 'GCHPchem',
+                            'SpeciesConcVV_AW03          ', 'GCHPchem',
+                            'SpeciesConcVV_AW04          ', 'GCHPchem',
+                            'SpeciesConcVV_AW05          ', 'GCHPchem',
+                            'SpeciesConcVV_AW06          ', 'GCHPchem',
+                            'SpeciesConcVV_AW07          ', 'GCHPchem',
+                            'SpeciesConcVV_AW08          ', 'GCHPchem',
+                            'SpeciesConcVV_AW09          ', 'GCHPchem',
+                            'SpeciesConcVV_AW10          ', 'GCHPchem',
+                            'SpeciesConcVV_AW11          ', 'GCHPchem',
+                            'SpeciesConcVV_AW12          ', 'GCHPchem',
+                            'SpeciesConcVV_AW13          ', 'GCHPchem',
+                            'SpeciesConcVV_AW14          ', 'GCHPchem',
+                            'SpeciesConcVV_AW15          ', 'GCHPchem',
+                            'SpeciesConcVV_RCOOH          ', 'GCHPchem',
```

Comment:
Adding these lines to the default HISTORY.rc.fullchem for GCHP causes non-TOMAS full-chemistry runs to crash because these TOMAS species are not defined. I have fixed this in the [bugfix/tomas-gchp](https://github.com/geoschem/geos-chem/tree/bugfix/tomas-gchp) branch in commit https://github.com/geoschem/geos-chem/commit/1fd5c26cde4243d7cf3a13656aa3311546648146 by removing these lines and adding them via a `sed` command in `setupConfigFiles.sh` which is called from `createRunDir.sh`.

Reply:
Thanks for catching this issue and for the code update!